.github/ISSUE_TEMPLATE/feature-request.yml

```
@@ -0,0 +1,40 @@
+name: "Feature request"
+description: "Create a feature request to help us improve"
+title: "[ENH] - <title>"
+labels: ["type: enhancement"]
+
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Hi! Thanks for using Nebari and taking some time to contribute to this project.
+
+  - type: textarea
+    attributes:
+      label: Feature description
+      description: |
+        Describe what you are proposing. Provide as much context as possible and link to related issues and/or pull requests.
+        This section should contain "what" you are proposing.
+        Are you having any problems? Briefly describe what your painpoints are. For example: "I'm always frustrated when ..."
```

```suggestion
  - type: textarea
    attributes:
      label: Context
      description: |
        Describe what you are proposing. Provide as much context as possible and link to related issues and/or pull requests.
        This section should contain "what" you are proposing.
        Are you having any problems? Briefly describe what your pain points are. 
```

removing this in favour of maintainers' sanity